pkgs/applications/misc/qtpass/default.nix

```diff
@@ -34,6 +34,9 @@ mkDerivation rec {
     })
   ];
 
+  # HACK `propagatedSandboxProfile` does not appear to actually propagate the sandbox profile from `qt5.qtbase`
+  sandboxProfile = qtbase.__sandboxProfile;
```

Comment:
Are we sure we don't want to use `qtbase.__propagatedSandboxProfile`?

Reply:
That makes sense. It makes it even more clear this is a workaround since the `__propagatedSandboxProfile` should be applied automatically.

Reply:
I pushed updated commits for qtpass and qtkeychain to use `__propagatedSandboxProfile`.